rep-0144.rst

```diff
@@ -102,7 +102,7 @@ Special Suffixes:
   ``pr2_robot``
 * a package containing the URDF and meshes of a robot should be named 
   ``<name_of_the_robot>_description``, e.g ``pr2_description``
-* if a package is meant for test only, prepend ``test``
+* if a package is meant for test only, append ``tests``
```

Comment:
If it's prepended I read it as the verb, ala test the package xyz,  while a appended it is a noun referring to a package which contains tests related to xyz package. As most test packages should have multiple tests if treated as a noun I'd expect it to be plural for a collection of tests.